modules/git/command.go

```diff
@@ -281,6 +281,7 @@ func (c *Command) Run(opts *RunOpts) error {
 		log.Debug("git.Command.RunDir(%s): %s", opts.Dir, c)
 	}
 
+	startTime := time.Now()
```

Comment:
Shouldn't it be right before `exec.CommandContext` to only measure how long the command took?

Reply:
Done.